src/screens/PluginStore/UploadPluginModal.tsx

```
@@ -105,196 +102,203 @@ const UploadPluginModal: React.FC<IUploadPluginModalProps> = ({
+            selectedFile ? 'border-primary bg-light' : 'border-secondary'
+          } ${styles.uploadArea}`}
+          onDragOver={handleDragOver}
+          onDrop={handleDrop}
+          onClick={() => fileInputRef.current?.click()}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter' || e.key === ' ') {
+              fileInputRef.current?.click();
+            }
+          }}
+          role="button"
+          tabIndex={0}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)